docs/_static/robots.txt

```diff
@@ -0,0 +1,91 @@
+Sitemap: http://docs.soliditylang.org/sitemap.xml
+
+# Prevent development and old documentation from showing up in search results.
```

Comment:
This comment seems to suggest that `/en/breaking/` is an old release. Or an I reading that wrong?

```suggestion
# Prevent development and breaking release documentation from showing up in search results.
```

Reply:
I'm not sure about the wording. Maybe we should say 

```suggestion
# Prevent documentation for older Solidity versions and the development branch from showing up in search results.
```

Reply:
Oh, so this comment probably applies to the stuff under `# Disallow old versions` too? It's above the `/en/develop/` + `/en/breaking/` part so it looks like it's describing only that.

In that case see my suggestion below.